### ISSUE TITLE
SOCKS proxy only accessible from host

### ISSUE THREAD

**Issue by psamim** [mook/docker-openvpn-client-socks#3]
Hi,

I can run the container successfully and it works fine from host but I cannot connect to it from other IPs on the same local network. 

Thanks,

**Comment by xorr0** [comment]
I too am experiencing the same issue. From the local machine things work fine, but from any other device on the local network I cannot get a response from the SOCKS5 proxy. I've made sure to open the firewall ports on 1080 tcp+udp, and I've tried many other ideas but nothing seems to work. Any suggestions?

**Comment by afdalwahyu** [comment]
my current implementation is using [gost](https://github.com/ginuerzh/gost) to forward socks from docker. btw i'm using this [dockerfile](https://github.com/kizzx2/docker-openvpn-client-socks)

`$> gost -L=:6666 -F socks4://127.0.0.1:1080`

where 127.0.0.1:1080 is address from container openvpn proxy, and you're running gost from your current os.

**Comment by mook** [comment]
That is correct; the [start](https://github.com/mook/docker-openvpn-client-socks/blob/bc28a5374f8ae57c05d0a5c8d4831dcacb623ead/start#L9) script has `--publish` on `127.0.0.1`.  If you want to make it work outside, start by changing that (but I haven't tried it recently, so it might not work).

**Comment by antoniozh** [comment]
Hi, I wrote my own script for this. 
```
#!/bin/bash

if [ ! -z $SUBNET ]; then
        ip route add "$SUBNET"/24 via 172.20.0.1 dev eth0  
        [ $? -eq 1 ] && echo Subnet "$SUBNET"/24 tunneled! 
        echo "Subnet script ran"
fi
```

Make sure to adjust the SUBNET variable and your gateway. You can find out the device ip by grepping but I forgot where I coded that
